hardware.inc

```diff
@@ -583,6 +584,8 @@ DEF rSCX EQU $FF43
 ; --
 DEF rLY EQU $FF44
 
+DEF LY_VBLANK EQU 144

```

Comment:
That's `SCRN_Y`. Is it worth creating a separate constant?

Reply:
Doubt it now that I'm thinking about it again

Reply:
Given that it's not obvious that it can be used as such, maybe adding a comment next to `SCRN_Y` could help nudging towards that?

Reply:
Yeah, that'd help with that particular instance of `SCRN_Y` usage